src/dashboard/ChartDashboardObject.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace luya\admin\dashboard;
+
+use Yii;
+use yii\helpers\Html;
+use yii\helpers\ArrayHelper;
+
+/**
+ * Fast generated Dashboard Objects.
+ *
+ * The default object is the default class for all {{luya\admin\base\Module::$dashboardObjects}} items without a class defintion.
```

Comment:
Please add an example of how to implement, for example and example response what the api must provide.